packages/salesforcedx-vscode-core/src/decorators/scratch-org-decorator.ts

```diff
@@ -10,10 +10,14 @@ import * as os from 'os';
 import * as path from 'path';
 import { StatusBarAlignment, StatusBarItem, window, workspace } from 'vscode';
 import { nls } from '../messages';
-import { getRootWorkspacePath, hasRootWorkspace } from '../util';
+import { workspaceUtils } from '../util';
 
-const CONFIG_FILE = hasRootWorkspace()
-  ? path.join(getRootWorkspacePath(), '.sfdx', 'sfdx-config.json')
+const CONFIG_FILE = workspaceUtils.hasRootWorkspace()
+  ? path.join(
+      workspaceUtils.getRootWorkspacePath(),
+      '.sfdx',
```

Comment:
Is it OK to use a path with `.sfdx` here, or is this another case where we need to call an API?

Reply:
If we stick with sfdx, then to match the other code this should be added:
`import { SFDX_PROJECT_FILE } from '../constants';`

Reply:
yup, ok for now.  We have a bunch of work items to address all these in the core v3 epic.